client/tests/test_client.py

```diff
@@ -85,3 +88,11 @@ def test_delete_clients(self):
         self.assertEqual(response.status_code, 302)
         self.assertEqual(Client.objects.count(), 0)
         self.api_client.logout()
+
+    def test_create_dummy_clients(self):
+        """Test created dummy clients command."""
+        out = StringIO()
+        # args = []
+        # opts = {'total': 15}
+        call_command('create_dummy_clients', 10, stdout=out)
```

Comment:
@Ayub-Khan can you help me out a little here?
The command `python manage.py create_dummy_clients 10` works fine on terminal.
But when I call from here like in the line or like this
```
call_command('create_dummy_clients', total=10, stdout=out)
```
Or even like this
```
call_command('create_dummy_clients', '--total=10', stdout=out)
```
it fails. Do you have any sample project where you have called the command in a test case?